src/plantuml.cpp

```diff
@@ -95,34 +73,153 @@ QCString PlantumlManager::writePlantUMLSource(const QCString &outDirArg,const QC
         case '\t': break;
         case ' ':  break;
         case '@':
-          if (initial && literal_at(p,"start")) // @start...
+          if (!insideComment && literal_at(p,"start")) // @start...
```

Comment:
I don't really understand the use-case of this inline parser stuff. Do you have an example?

Reply:
The example: [example.tar.gz](https://github.com/user-attachments/files/19525459/example.tar.gz)